jsapp/js/components/modalForms/connectProjects.es6

```diff
@@ -186,163 +298,293 @@ class ConnectProjects extends React.Component {
+          }
+
+          {!this.state.isLoading && this.state.attachedParents.length > 0 &&
+            this.state.attachedParents.map((item, n) => {
+              return (
+                <li key={n} className='connect-projects__import--list-item'>
+                  <i className="k-icon k-icon-check"/>
+
+                  <div className='connect-projects__import--labels'>
+                    <span className='connect-projects__import--labels--filename'>
+                      {item.filename}
+                    </span>
+
+                    <span className='connect-projects__import--labels--parent'>
+                      {item.parentName}
+                    </span>
+                  </div>
+
+                  <div className='connect-projects__import--options'>
+                    <i
+                      className="k-icon k-icon-trash"
+                      onClick={() => this.onRemoveAttachment(item.attachmentUrl)}
+                    />
+
+                    <i
+                      className="k-icon k-icon-settings"
+                      onClick={() => this.showColumnFilterModal(
+                        this.props.asset,
+                        {
+                          uid: item.parentUid,
+                          name: item.parentName,
+                          url: item.parentUrl,
+                        },
+                        item.filename,
+                        item.childFields,
+                        item.attachmentUrl,
+                      )}
+                    />
+                  </div>
+                </li>
+              );
+            })
+          }
+        </ul>
+      </div>
+    );
   }
 
   render() {
     return (
-      <bem.FormModal__form
-        className='connect-projects'
-        onSubmit={this.confirmAttachment}
-      >
+      <bem.FormView__row>
         {/* Enable data sharing */}
-        <bem.FormModal__item m='data-sharing'>
-          <div className='connect-projects__header'>
-            <i className="k-icon k-icon-folder-out"/>
-            <h2>{t('Share data with other project forms')}</h2>
-          </div>
-          <p>
-            {t('Enable data sharing to allow other forms to import and use dynamic data from this project. Learn more about dynamic data attachments')}
-            &nbsp;
-            <a href='#'>{t('here')}</a>
-          </p>
-          {this.renderSwitchLabel()}
-        </bem.FormModal__item>
+        <bem.FormView__cell m={['page-title']}>
+          <i className="k-icon k-icon-folder-out"/>
+          <h2>{t('Share data with other project forms')}</h2>
+        </bem.FormView__cell>
+
+        <bem.FormView__cell m={['box', 'padding']}>
+          <bem.FormView__form>
+            <span>
+              {t('Enable data sharing to allow other forms to import and use dynamic data from this project. Learn more about dynamic data attachments')}
+              &nbsp;
+              <a href='#'>{t('here')}</a>
+            </span>
```

Comment:
The mockups had two because I think we want two separate help pages for the two different actions a user can take here, please correct me if I'm wrong @Ig-Rebollo !